src/lib/components/LogoNNS.svelte

```diff
@@ -1,126 +1,127 @@
+    />
+    <path
+      d="M77.4734 31.9472H72.1871V23.7278H77.4503V25.6058H74.3434V26.9042H77.172V28.7127H74.3434V30.0343H77.4734V31.9472Z"
+    />
+    <path
+      d="M70.3525 31.9472H68.1266L65.2748 27.0665V31.9472H63.1301V23.7278H65.6341L68.1962 28.1911V23.7278H70.3525V31.9472Z"
+    />
+    <path
+      d="M59.7364 23.7278L56.7107 27.4607L59.6901 31.9472H57.0237L55.2964 29.1996L54.5428 30.1387V31.9472H52.3749V23.7278H54.5428V26.9622L57.0701 23.7278H59.7364Z"
+    />
+    <path
+      d="M46.8894 31.9472H44.7331V23.7278H48.1994C49.8688 23.7278 50.9585 24.8407 50.9585 26.371C50.9585 27.4839 50.3441 28.307 49.4398 28.6664L50.9933 31.9472H48.6399L47.3415 28.9678H46.8894V31.9472ZM46.8894 27.2984H47.7936C48.4428 27.2984 48.779 26.9274 48.779 26.4057C48.779 25.8841 48.4428 25.5247 47.7936 25.5247H46.8894V27.2984Z"
+    />
+    <path
+      d="M37.1558 27.8317C37.1558 29.3504 38.1759 30.0343 39.1382 30.0343C40.1004 30.0343 41.1205 29.3504 41.1205 27.8317C41.1205 26.3362 40.1004 25.6522 39.1382 25.6522C38.1759 25.6522 37.1558 26.3362 37.1558 27.8317ZM34.9183 27.8433C34.9183 25.1769 36.8775 23.5539 39.1382 23.5539C41.3988 23.5539 43.358 25.1769 43.358 27.8433C43.358 30.5096 41.3988 32.1327 39.1382 32.1327C36.8775 32.1327 34.9183 30.5096 34.9183 27.8433Z"
+    />
+    <path
+      d="M31.0796 28.5968L32.1809 23.7278H34.4067L32.3084 31.9472H30.0246L28.5639 26.9622L27.1032 31.9472H24.773L22.6747 23.7278H24.9701L26.0946 28.6548L27.5437 23.7278H29.6304L31.0796 28.5968Z"
+    />
+    <path
+      d="M22.2032 25.7797H19.8267V31.9472H17.6472V25.7797H15.2822V23.7278H22.2032V25.7797Z"
+    />
+    <path
+      d="M14.4142 31.9472H9.12787V23.7278H14.3911V25.6058H11.2842V26.9042H14.1128V28.7127H11.2842V30.0343H14.4142V31.9472Z"
+    />
+    <path
+      d="M7.29331 31.9472H5.06747L2.21561 27.0665V31.9472H0.0709229V23.7278H2.57499L5.13703 28.1911V23.7278H7.29331V31.9472Z"
+    />
+  </g>
+  <defs>
+    <linearGradient
+      id="paint0_linear_1386_9727"
+      x1="53.2309"
+      y1="18.5964"
+      x2="39.53"
+      y2="4.40876"
+      gradientUnits="userSpaceOnUse"
+    >
+      <stop offset="0.22" stop-color="#FF0079" />
+      <stop offset="0.89" stop-color="#592489" />
+    </linearGradient>
+    <linearGradient
+      id="paint1_linear_1386_9727"
+      x1="64.1309"
+      y1="1.50728"
+      x2="77.8311"
+      y2="15.6949"
+      gradientUnits="userSpaceOnUse"
+    >
+      <stop offset="0.21" stop-color="#FF4B00" />
+      <stop offset="0.68" stop-color="#FFAB00" />
+    </linearGradient>
+    <clipPath id="clip0_1386_9727">
+      <rect
+        width="41.8421"
+        height="19.7368"
+        fill="white"
+        transform="translate(37.8572)"
+      />
+    </clipPath>
+  </defs>
+</svg>
 
 <style lang="scss">
-  div {
-    display: flex;
-    justify-content: center;
+  @use "../styles/mixins/media";
+
+  svg {
+    padding: 0 var(--padding-6x) 0;
```

Comment:
I don't think this belongs in the image. Can we add rather the paddings where the image is used?

Reply:
Done.